Mastodon/Scene/Search/Search/SearchViewController.swift

```diff
@@ -73,19 +73,19 @@ extension SearchViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        setupBackgroundColor(theme: ThemeService.shared.currentTheme.value)
+        setupAppearance(theme: ThemeService.shared.currentTheme.value)
         ThemeService.shared.currentTheme
             .receive(on: DispatchQueue.main)
             .sink { [weak self] theme in
                 guard let self = self else { return }
-                self.setupBackgroundColor(theme: theme)
+                self.setupAppearance(theme: theme)
```

Comment:
Is there a benefit to calling this function twice, or would calling it in the Sink be enough?

Reply:
I don't think the sink is called unless the theme changes; the initial call is needed to set things up with the current theme. (That's the precedence I’ve seen in the code base, I didn't actually check though)

Reply:
That makes sense! I'm still familiarizing myself with Combine and the concept of disposable bags, so excuse my ignorance!